config/client_config.go

```diff
@@ -43,7 +43,7 @@ type ProviderConfig struct {
 	Validator           *ValidatorConfig
 	PassthroughPatterns []string
 	FrontingSNIs        map[string]*fronted.SNIConfig
-	VerifyHostname      *string
+	VerifyHostname      *string `yaml:"verifyHostname,omitempty"`
```

Comment:
Adding context from Slack for posterity: @WendelHime added tests, originally as part of an effort to make a change to the default provider ID.  Though we did not end up changing the default provider ID, Wendel's tests caught a bug here.  The issue is that the YAML parser will always set this value to nil, unless `omitempty` is configured.  This is true even if `verifyHostname` is present in the YAML.  The bug does not present itself currently, but will if we try to use `genconfig` with this new field.

Is this correct, @WendelHime?

Reply:
Yes, that's it!